mne/viz/_brain/_timeviewer.py

```diff
@@ -479,26 +490,49 @@ def __init__(self, brain, show_traces=False):
             self._mouse_no_mvt = -1
             self.enable_point_picking()
 
-        # remove default picking menu
+        self.update_menu()
+
+    def update_menu(self):
         main_menu = self.plotter.main_menu
+        file_menu = None
+
+        # add help menu
+        help_menu = main_menu.addMenu('Help')
+        help_menu.addAction('Show MNE key bindings', self.help, '?')
+
+        # remove default picking menu
         to_remove = list()
         for action in main_menu.actions():
             if action.text() == "Tools":
                 to_remove.append(action)
+            elif action.text() == "File":
+                file_menu = action.menu()
         for action in to_remove:
             main_menu.removeAction(action)
-
-        # setup key bindings
-        self.key_bindings = {
-            '?': self.help,
-            'i': self.toggle_interface,
-            's': self.apply_auto_scaling,
-            'r': self.restore_user_scaling,
-            'c': self.clear_points,
-            ' ': self.toggle_playback,
-        }
-        menu = self.plotter.main_menu.addMenu('Help')
-        menu.addAction('Show MNE key bindings\t?', self.help)
+        to_remove.clear()
+
+        # order the file menu
+        if file_menu is not None:
+            for action in file_menu.actions():
+                if action.text() == "Take Screenshot":
+                    movie_action = file_menu.addAction(
+                        'Save movie',
+                        self.save_movie,
+                        "ctrl+shift+s"
+                    )
+                    # insert at the right place
+                    file_menu.insertAction(action, movie_action)
+                    break
```

Comment:
When I actually trigger the action, the window freezes for a while. Is it possible to do one (preferably all) of:

- turn the cursor to the hourglass/waiting one for the Qt window until the operation completes
- have a status bar update like "Saving movie ..." while operating, and then clear when done

I think both are useful to look into, as status bars are nice for all sorts of updates. Both of these should hopefully be fairly easily doable in Qt with a few lines...?

Reply:
The main window does not have a status bar by default and using list comprehension in `_iter_time` does not allow progress bar.

Reply:
> The main window does not have a status bar by default and using list comprehension in _iter_time does not allow progress bar.

My suggestion would be to add it (we'll probably want it for other stuff at some point anyway), and don't use list comprehension but a two- or three-line loop instead. But we can do these incrementally, feel free to add to the tracking issue instead

Reply:
> My suggestion would be to add it 

That's what I did in b86dbc9

> don't use list comprehension but a two- or three-line loop instead

:+1: I'll add it to the list. For now only the `Saving movie ...` label is available